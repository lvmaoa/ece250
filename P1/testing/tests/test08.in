m 7
push_front "news" "news.google.com"
push_back "google" "www.google.com"
push_front "stackexchange" "www.stackexchange.com"
print
size
pop_front
print
size
pop_back
print
size
push_back "bored" "www.boredpanda.com"
print
size
push_back "uw" "www.uwaterloo.ca"
push_front "piazza" "www.piazza.com"
pop_front
size
print
exit

