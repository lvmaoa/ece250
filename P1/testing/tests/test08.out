success
success
success
success
"google" "www.google.com"
"news" "news.google.com"
"stackexchange" "www.stackexchange.com"
size is 3
success
"google" "www.google.com"
"news" "news.google.com"
size is 2
success
"news" "news.google.com"
size is 1
success
"bored" "www.boredpanda.com"
"news" "news.google.com"
size is 2
success
success
success
size is 3
"uw" "www.uwaterloo.ca"
"bored" "www.boredpanda.com"
"news" "news.google.com"
